.github/workflows/unit_tests.yml

```diff
@@ -0,0 +1,49 @@
+name: unit-tests
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.id }}
+  cancel-in-progress: true
+
+env:
+  OPENAI_API_KEY: dummy
+  TELEMETRY_ENABLED: false
+  REFLEX_DEP: "git+https://github.com/reflex-dev/reflex@main"
+
+on:
+  push:
+    branches: ['main']
+    paths-ignore:
```

Comment:
Why are we ignoring markdown files for this GitHub workflow? As these can also contain valid or invalid urls.